### PR TITLE
 Ensure necessary embed strings are present for each target

### PR DESCRIPTION
**Pull request by AlCutter**
This PR ensures that the Makefile will complain if required keys are not provided,

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -147,6 +158,10 @@ $(APP).elf: check_tamago
 
 
 $(APP)_manifest:
+	@if [ "${APPLET_PRIVATE_KEY}" == "" ] || [ ! -f "${APPLET_PRIVATE_KEY}" ]; then \
+		echo 'You need to set the APPLET_PRIVATE_KEY variable to a valid note verifier key path'; \
```
Comment (mhutchinson) [inline Makefile:162]:
```suggestion
		echo 'You need to set the APPLET_PRIVATE_KEY variable to a valid note signing key path'; \
```

I know you're just testing how closely I'm reviewing it :-)

**Review by mhutchinson — APPROVED** [review]

File: Makefile (hunk 1 above)
Comment (AlCutter) [reply Makefile]: Exactly, well done - good job :)

File: Dockerfile [hunk 2]
```diff
@@ -1,9 +1,9 @@
 FROM golang:1.21-bookworm
 
 ARG TAMAGO_VERSION
-ARG FT_LOG_ORIGIN
```
Comment (mhutchinson) [inline Dockerfile:4]: What's the motivation behind renaming this arg but not the URL and bin?

**Review by mhutchinson — APPROVED** [review]

File: Dockerfile (hunk 2 above)
Comment (AlCutter) [reply Dockerfile:4]: Other repos call this `LOG_ORIGIN` so I wanted to try to have common vars be named the same across all repos. Can remove the `FT_` prefix on the others if you want in a follow up?

**Pull request merged**